extensions/wikia/AdEngine/js/provider/directGpt.js

```diff
@@ -7,10 +7,9 @@ define('ext.wikia.adEngine.provider.directGpt', [
 	'ext.wikia.adEngine.slot.adUnitBuilder',
 	'ext.wikia.adEngine.slot.service.kiloAdUnitBuilder',
 	'ext.wikia.adEngine.slotTweaker',
-	require.optional('ext.wikia.adEngine.lookup.openx.openXBidderHelper'),
 	require.optional('ext.wikia.aRecoveryEngine.pageFair.recovery'),
 	require.optional('ext.wikia.aRecoveryEngine.sourcePoint.recovery')
-], function (adContext, uapContext, factory, adUnitBuilder, kiloAdUnitBuilder, slotTweaker, openXHelper, pageFair, sourcePoint) {
+], function (adContext, uapContext, factory, adUnitBuilder, kiloAdUnitBuilder, slotTweaker, pageFair, sourcePoint) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 8 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)